src/components/custom-sdk/template/HMRC_ODX_GDSCheckAnswersPage/index.tsx

```diff
@@ -187,7 +192,7 @@ export default function HmrcOdxGdsCheckAnswersPage(props: HmrcOdxGdsCheckAnswers
       const originalLink = cloneLink;
       if (originalLink) {
         cloneLink.addEventListener('click', event => {
```

Comment:
Should we add any error handling here as its new functionality I think that we should handle any errors ie 
try {
 if (isCHBJourney() || isEduStartJourney()...
 else {
 const stepID = ...
 }
 } catch (error){
// display error message etc
}